vertexai/snippets/src/main/java/vertexai/gemini/QuestionAnswer.java

```diff
@@ -16,10 +16,12 @@
 
 package vertexai.gemini;
 
+// [START generativeaionvertexai_non_stream_text_basic]
 import com.google.cloud.vertexai.VertexAI;
 import com.google.cloud.vertexai.api.GenerateContentResponse;
 import com.google.cloud.vertexai.generativeai.GenerativeModel;
 import com.google.cloud.vertexai.generativeai.ResponseHandler;
+// [END generativeaionvertexai_non_stream_text_basic]
```

Comment:
Kindly remove this region tag and the one in line 39. It's okay and usual practice to include the full sample.

Reply:
@Sita04 this is ready for a re-review I think